### ISSUE TITLE
Facing an issues when upload build with releases notes param

### ISSUE THREAD

**Issue by Maxwell6635** [cianru/huawei-appgallery-publish-gradle-plugin#43]
huaweiPublish {
    instances {
        stagingRelease {
            credentialsPath = "$rootDir/huawei-credentials.json"
            deployType = "draft"
            buildFormat = "apk"
            publishTimeoutMs = 15_000
            publishPeriodMs = 3_000
            releaseTime = "2022-12-25T09:00:00+0800"
            releaseNotes = [
                    new ru.cian.huawei.publish.ReleaseNote(
                            "uk",
                            "$projectDir/release-notes-en.txt"
                    )
            ]
        }
    }
}

* What went wrong:
Execution failed for task ':app:publishHuaweiAppGalleryStagingRelease'.
> Update Release Notes for 'uk' is failed. Response: UpdateReleaseNotesResponse(ret=Ret(code=204144649, msg=[cds]update app information failed, additional msg is [appName cant be empty.]))

**Comment by cosic** [comment]
@Maxwell6635 Hi. As I sow from [API documentations](https://developer.huawei.com/consumer/en/doc/development/AppGallery-connect-References/agcapi-language-info-update-0000001158245057) 
> `appName` - App name in a language. This parameter is mandatory when a language type is added.

you doesn't have description for new `uk` country in your Developer Console yet. Please create it once manualty and try again.

**Comment by Maxwell6635** [comment]
@cosic How should I add the app name for this plugins ?
And right now this configuration is set in console.
<img width="1261" alt="Screenshot 2022-12-22 at 5 47 22 PM" src="https://user-images.githubusercontent.com/12591474/209106641-ee3069f6-6a88-4db0-911c-9ae072953b39.png">

**Comment by cosic** [comment]
May you have accidentally confused `uk` (Ukrainian) and `en-GB` (English - UK)? Please see [languages doc](https://developer.huawei.com/consumer/en/doc/development/AppGallery-connect-References/agcapi-reference-langtype-0000001158245079)

**Comment by Maxwell6635** [comment]
@cosic Hey, thanks for the info man. Let me try again.

**Comment by Maxwell6635** [comment]
@cosic It working right now, thanks man.
